src/encoded/renderers.py

```diff
@@ -196,20 +209,6 @@ def security_tween(request):
 
         return handler(request)
 
-        # This was commented out when we introduced JWT authentication
```

Comment:
This is just getting rid of code that's long been commented out and is not believed to be needed now.